.strict-typing

```diff
@@ -155,7 +155,6 @@ homeassistant.components.shelly.*
 homeassistant.components.simplisafe.*
 homeassistant.components.slack.*
 homeassistant.components.smhi.*
-homeassistant.components.sonos.media_player
```

Comment:
`homeassistant.components.sonos.media_player` has mypy errors, which were hidden by the fact that `homeassistant.components.sonos.*` as a whole was in IGNORED_MODULES.